eq-author/src/App/section/Design/index.test.js

```diff
@@ -465,14 +468,15 @@ describe("SectionRoute", () => {
       });
     });
 
-    it("should enable the preview tab when the introduction is enabled", async () => {
+    it("should enable the preview tab when introduction is enabled and introduction title or content is not empty", async () => {
```

Comment:
Should this not also be checking for introductionPageDescription? (and in other tests below)